docs/whats-new.md

```diff
@@ -4,7 +4,7 @@
 
 Release date: TBD
 
-* NEW: `getProjectionMatrix` can match the behavior of the latest Mapbox (when called with `nearZMultipler: 1 / viewport.height, farZMultiplier: viewport.height`)
+* NEW: `getProjectionMatrix` can match the behavior of the latest Mapbox (when called with `nearZMultipler: 1 / viewport.height, farZMultiplier: 1`)
```

Comment:
Perhaps time to start adopting a more feature based style instead of CHANGELOG like notes?

**z-buffer alignment** - `getProjectionMatrix` can now match `near`/`far` range of the projection matrix used with the mapbox custom layers API. This enables aligned Z buffer values, which means that 3D geometries drawn into a mapboxgl WebGL context will be correctly mixed with mapbox-rendered 3D geometry."